### PR TITLE
add blk meta schema

### PR DESCRIPTION
**Pull request by aptend**
## What type of PR is this?

- [ ] API-change
- [ ] BUG
- [ ] Improvement
- [ ] Documentation
- [x] Feature
- [ ] Test and CI
- [ ] Code Refactoring

## Which issue(s) this PR fixes:

issue #

## What this PR does / why we need it:

add block meta schema to catalog package

### CODE REVIEW

**Review by XuPeng-SH — APPROVED** [review]
lgtm

File: pkg/catalog/catalog.go [hunk 1]
```diff
@@ -123,6 +123,20 @@ func ParseEntryList(es []*api.Entry) (any, []*api.Entry, error) {
 	}
 }
 
+// util function, it will be used later
```
Comment (nnsgmsone) [inline pkg/catalog/catalog.go:126]: Why comments and not the actual code?

Comment (aptend) [reply pkg/catalog/catalog.go]: a dummy test is added to use it

**Review by nnsgmsone — APPROVED** [review]

**Review by XuPeng-SH — APPROVED** [review]
lgtm

**Pull request merged**